Fail to build Quarkus in master because of minikube dependency

`./mvnw clean install`

```
Caused by: java.lang.RuntimeException: Unable to start Quarkus test resource io.quarkus.test.kubernetes.client.KubernetesMockServerTestResource@3676ac27
        at io.quarkus.test.common.TestResourceManager.start(TestResourceManager.java:47)
        at io.quarkus.test.junit.QuarkusTestExtension.ensureStarted(QuarkusTestExtension.java:179)
        at io.quarkus.test.junit.QuarkusTestExtension.beforeAll(QuarkusTestExtension.java:208)
        at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.lambda$invokeBeforeAllCallbacks$7(ClassBasedTestDescriptor.java:359)
        at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
        at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.invokeBeforeAllCallbacks(ClassBasedTestDescriptor.java:359)
        at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.before(ClassBasedTestDescriptor.java:189)
        at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.before(ClassBasedTestDescriptor.java:78)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$5(NodeTestTask.java:132)
        ... 31 more
Caused by: io.fabric8.kubernetes.client.KubernetesClientException: An error has occurred.
        at io.fabric8.kubernetes.client.KubernetesClientException.launderThrowable(KubernetesClientException.java:64)
        at io.fabric8.kubernetes.client.KubernetesClientException.launderThrowable(KubernetesClientException.java:53)
        at io.fabric8.kubernetes.client.utils.HttpClientUtils.createHttpClient(HttpClientUtils.java:199)
        at io.fabric8.kubernetes.client.utils.HttpClientUtils.createHttpClientForMockServer(HttpClientUtils.java:69)
        at io.fabric8.kubernetes.client.server.mock.KubernetesMockServer.createClient(KubernetesMockServer.java:86)
        at io.quarkus.test.kubernetes.client.KubernetesMockServerTestResource.start(KubernetesMockServerTestResource.java:22)
        at io.quarkus.test.common.TestResourceManager.start(TestResourceManager.java:45)
        ... 39 more
Caused by: java.nio.file.NoSuchFileException: /Users/emmanuel/.minikube/client.crt
        at java.base/sun.nio.fs.UnixException.translateToIOException(UnixException.java:92)
        at java.base/sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:111)
        at java.base/sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:116)
        at java.base/sun.nio.fs.UnixFileSystemProvider.newByteChannel(UnixFileSystemProvider.java:219)
        at java.base/java.nio.file.Files.newByteChannel(Files.java:370)
        at java.base/java.nio.file.Files.newByteChannel(Files.java:421)
        at java.base/java.nio.file.Files.readAllBytes(Files.java:3205)
        at io.fabric8.kubernetes.client.internal.CertUtils.getInputStreamFromDataOrFile(CertUtils.java:72)
        at io.fabric8.kubernetes.client.internal.CertUtils.createKeyStore(CertUtils.java:242)
        at io.fabric8.kubernetes.client.internal.SSLUtils.keyManagers(SSLUtils.java:128)
        at io.fabric8.kubernetes.client.internal.SSLUtils.keyManagers(SSLUtils.java:122)
        at io.fabric8.kubernetes.client.utils.HttpClientUtils.createHttpClient(HttpClientUtils.java:85)
        ... 43 more
```

We need to list it as a required dependency and point to it.

--- a/test-framework/kubernetes-client/src/main/java/io/quarkus/test/kubernetes/client/KubernetesMockServerTestResource.java
+++ b/test-framework/kubernetes-client/src/main/java/io/quarkus/test/kubernetes/client/KubernetesMockServerTestResource.java
@@ -15,17 +15,17 @@ public class KubernetesMockServerTestResource implements QuarkusTestResourceLife
 
     @Override
     public Map<String, String> start() {
-        mockServer = new KubernetesMockServer(useHttps());
-        mockServer.init();
-
         final Map<String, String> systemProps = new HashMap<>();
-        try (NamespacedKubernetesClient client = mockServer.createClient()) {
-            systemProps.put(Config.KUBERNETES_MASTER_SYSTEM_PROPERTY, client.getConfiguration().getMasterUrl());
-        }
         systemProps.put(Config.KUBERNETES_TRUST_CERT_SYSTEM_PROPERTY, "true");
         systemProps.put(Config.KUBERNETES_AUTH_TRYKUBECONFIG_SYSTEM_PROPERTY, "false");
         systemProps.put(Config.KUBERNETES_AUTH_TRYSERVICEACCOUNT_SYSTEM_PROPERTY, "false");
         systemProps.put(Config.KUBERNETES_NAMESPACE_SYSTEM_PROPERTY, "test");
+
+        mockServer = new KubernetesMockServer(useHttps());
+        mockServer.init();
+        try (NamespacedKubernetesClient client = mockServer.createClient()) {
+            systemProps.put(Config.KUBERNETES_MASTER_SYSTEM_PROPERTY, client.getConfiguration().getMasterUrl());
+        }
 
         return systemProps;
     }

